src/engine/renderer/tr_image.cpp

```diff
@@ -1702,6 +1703,22 @@ static void R_LoadImage( const char **buffer, byte **pic, int *width, int *heigh
 		std::string altName = Str::Format("%s.%s", filename, imageLoaders[i].ext);
 		const FS::PakInfo* pak = FS::PakPath::LocateFile(altName);
 
+		// Darkplaces or Doom3 packages can ship alternative texture path in the form of
+		//   dds/<path without ext>.dds
+		if ( pak == nullptr && !Q_stricmp("dds", imageLoaders[i].ext) )
```

Comment:
No need to use `imageLoaders[i].ext` here since you already know it's dds.

Reply:
we don't have to look for `dds/` prefixes for other formats.

Reply:
Oops, I meant to write that on the `altName = ...` line.

Reply:
This whole part is now obsoleted by last commit